extensions/wikia/Helios/HelperController.class.php

```diff
@@ -52,7 +56,7 @@ public function updateAntiSpoof()
 
 		if ( !empty( $this->wg->EnableAntiSpoofExt ) ) {
 			$oSpoofUser = new \SpoofUser( $sName );
-			$oSpoofUser->record();
+			return $this->response->setVal( 'success', $oSpoofUser->record() );
```

Comment:
`SpoofUser::record()` returns true even on failure, so the check for success might be inconclusive and misleading.